modules/jolt_physics/shapes/jolt_custom_decorated_shape.h

```diff
@@ -55,6 +55,8 @@ class JoltCustomDecoratedShape : public JPH::DecoratedShape {
 
 	virtual JPH::MassProperties GetMassProperties() const override { return mInnerShape->GetMassProperties(); }
 
+	virtual const JPH::PhysicsMaterial *GetMaterial(const JPH::SubShapeID &p_sub_shape_id) const override { return mInnerShape->GetMaterial(p_sub_shape_id); }
```

Comment:
This is the exact same implementation as JPH::DecoratedShape::GetMaterial, I think this can be left out.